py3status/docstrings.py

```diff
@@ -111,7 +111,7 @@ def create_readme(data):
 re_from_item = re.compile('^\s+-')
 re_from_data = re.compile('^@(author|license|source)($|[ \t])')
 re_from_tag = re.compile('((`[^`]*`)|[<>&])')
-re_from_defaults = re.compile('(\(default.*\))')
+re_from_defaults = re.compile('(\(default.*\))$')
```

Comment:
We do something a little more clever? but it quickly becomes unreadable :)

```
import re

r1 = re.compile('(\(default.*\))$')


r2 = re.compile('(\(default ('
               '[^(\'"]*'
               '|'
               '\([^(]*\)' # tuple
               '|'
               r'("(?:[^"\\]|\\.)*")'  # double quoted string
               '|'
               r"|('(?:[^'\\]|\\.)*')"  # single quoted string
               ')\))')

r3 = re.compile('(\(default ('
               '[^(\'"]*'
               '|'
               '\((' # tuple start
               '[^(\'"]*'
               '|'
               r'("(?:[^"\\]|\\.)*")'  # double quoted string
               '|'
               r"|('(?:[^'\\]|\\.)*')"  # single quoted string
               ')*\)' # tuple end
               '|'
               r'("(?:[^"\\]|\\.)*")'  # double quoted string
               '|'
               r"|('(?:[^'\\]|\\.)*')"  # single quoted string
               ')\))')


items = [
    '(default "moo")',
    '(default "moo") ',
    '(default ("moo"))',
    '(default ("m(oo"))',
    '(default ("m(oo", \'moo\'))',
    '(default ("moo",)',
    '(default ("mo)o",)',
    '(default "mo(o")',
]

for item in items:
    p1 = bool(r1.match(item))
    p2 = bool(r2.match(item))
    p3 = bool(r3.match(item))
    print(item, p1, p2, p3)
```

gives

```
(default "moo") True True True
(default "moo")  False True True
(default ("moo")) True True True
(default ("m(oo")) True False True
(default ("m(oo", 'moo')) True False True
(default ("moo",) True False False
(default ("mo)o",) True False False
(default "mo(o") True True True
```

We should get a proper parser going